pyrometheus/codegen/python.py

```diff
@@ -427,7 +427,7 @@ def get_fwd_rate_coefficients(self, temperature, concentrations):
         ones = self._pyro_zeros_like(temperature) + 1.0
         k_fwd = [
         %for react in sol.reactions():
-        %if isinstance(react, ct.FalloffReaction):
+        %if react.equation in [r.equation for _, r in falloff_reactions]:
```

Comment:
This idiom is kind of inefficient. It seems like we could just test `react` instead?

Reply:
That was my initial approach but found that, for example, for the San Diego yaml in `mechs/test`:

`sol.reaction(9).equation == falloff_reactions[0].equation`

is true, so the 10th reaction is Falloff, and it is the first in the `falloff_reactions` list, but

`sol.Reaction(9) == falloff_reactions[0]`

is false, so testing `react in falloff_reactions` does not help here. I haven't been able to pin down what specific property makes them appear different upon comparison.